ISSUE_TEMPLATE/chore.md

```diff
@@ -0,0 +1,9 @@
+### Chore
+
+**Description**
+
+Please describe the task you want to update.  
```

Comment:
task is also not about updating. Please extrapolate from reviews :)

Reply:
I am really sorry for the confusion.